xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/Applications.xml

```diff
@@ -232,21 +232,22 @@
 #panelheader($services.localization.render('panels.applications.title'))
 
 {{html}}
-&lt;ul class="applicationsPanel nav nav-pills nav-stacked"&gt;
```

Comment:
Why did you remove these classes?

Reply:
Because removing them did not change significantly the presentation, and they brought an unexpected style from bootstrap 3 that would make the color match badly on some color themes. 
On Darkly, the child anchors were just white instead of being green like any other anchor in the panels, and almost anywhere on the page vv
![20843-removeBootstrapNav](https://github.com/xwiki/xwiki-platform/assets/28761965/92824037-2367-41d5-84b8-a9271e29efe0)
Moreover I think this reduces the dependency on Bootstrap 3, which is a good thing if we want to move on from it someday. :)

Reply:
Looking at this screenshot, the indentation of the "More applications" seems changed. Previously, this was aligned with the other entries like "Dashboard". Why has this alignment changed?